app/controllers/orga/teams_controller.rb

```diff
@@ -78,9 +77,9 @@ def team_params
       :name, :twitter_handle, :github_handle, :description, :post_info, :event_id,
       :checked, :'starts_on(1i)', :'starts_on(2i)', :'starts_on(3i)',
       :'finishes_on(1i)', :'finishes_on(2i)', :'finishes_on(3i)', :invisible,
-      :season_id, :kind, :project_name,
+      :project_name, :season_id, :kind,
+      conference_preference_attributes: [:id, :first_conference_id, :second_conference_id, :lightning_talk, :comment, :_destroy],
```

Comment:
When switching to using a validation for the two `terms` fields, they have to be permitted here as well of course.